OpenRA.Game/Map/TileSet.cs

```diff
@@ -185,7 +185,6 @@ public class TileSet
 		public readonly string Id;
 		public readonly int SheetSize = 512;
 		public readonly string Palette;
```

Comment:
The same applies here too.

Reply:
https://github.com/Mailaender/OpenRA/tree/move-palette-from-tileset will be a lot more invasive. I probably won't finish it.

Reply:
Ok, lets focus on just `PlayerPalette` here then.